mpas_analysis/ocean/plot_climatology_map_subtask.py

```diff
@@ -550,6 +553,77 @@ def _plot_antarctic(self, remappedModelClimatology,
         x = interp_extrap_corner(remappedModelClimatology['x'].values)
         y = interp_extrap_corner(remappedModelClimatology['y'].values)
 
+        filePrefix = self.filePrefix
+        outFileName = '{}/{}.png'.format(self.plotsDirectory, filePrefix)
+        title = '{} ({}, years {:04d}-{:04d})'.format(
+                self.fieldNameInTitle, season, self.startYear,
+                self.endYear)
+
+        plot_polar_projection_comparison(
+            config,
+            x,
+            y,
+            self.landMask,
+            modelOutput,
+            refOutput,
+            bias,
+            fileout=outFileName,
+            colorMapSectionName=configSectionName,
+            title=title,
+            modelTitle='{}'.format(mainRunName),
+            refTitle=self.refTitleLabel,
+            diffTitle=self.diffTitleLabel,
+            cbarlabel=self.unitsLabel)
+
+        upperGridName = comparisonGridName[0].upper() + comparisonGridName[1:]
+        caption = '{} {}'.format(season, self.imageCaption)
+        write_image_xml(
+            config,
+            filePrefix,
+            componentName='Ocean',
+            componentSubdirectory='ocean',
+            galleryGroup='{} {}'.format(upperGridName,
+                                        self.galleryGroup),
+            groupSubtitle=self.groupSubtitle,
+            groupLink=self.groupLink,
+            gallery=self.galleryName,
+            thumbnailDescription=self.thumbnailDescription,
+            imageDescription=caption,
+            imageCaption=caption)
+
+        # }}}
+
+    def _plot_arctic(self, remappedModelClimatology,
```

Comment:
@milenaveneziani, would it make sense to combine `_plot_antarctic()` and `_plot_arctic()` into a single function?  They seem nearly identical.

Reply:
yes, I noticed that too. How about `_plot_polar` or maybe `_plot_polar_stereographic`?

Reply:
Yep, either of those if perfect.